src/spec-common/injectHeadless.ts

```diff
@@ -307,6 +315,9 @@ export async function setupInContainer(params: ResolverParameters, containerProp
 	if (params.postCreate.enabled) {
 		await runPostCreateCommands(params, containerProperties, updatedConfig, remoteEnv, false);
 	}
+	if (params.dotfilesConfiguration) {
```

Comment:
Changed the order, I found confusing that its before the personalization check instead of after?

Reply:
A `~/.gitconfig` from dotfiles would win over a `~/.gitconfig` from the local machine in the Dev Containers extension (which skips copying the local one when there already is one in the container). Maybe it could argued that it should be the other way around, this is just the current order. Ideally we can keep it this way to allow for adoption without breaking existing setups.